tests/test.py

```diff
@@ -1146,14 +1146,34 @@ def run_terraform(*, image: str, user: str) -> None:
         (
             {
                 "DISABLE_HOOKS": "false",
-                "AUTODETECT": "true",
```

Comment:
Why?

Reply:
It would throw an error about no tf files with auto detect enabled and exit 1 before it got to my hook, which expected 230. I only needed one failure to test learning mode, so for consistency, I removed it from all three tests.